LiskHQ/lisk
# Rollback version patching mechanism - Closes #2466

### What was the problem?

Our approach to versioning should be compatible with older nodes.
A previous incomplete patch needed to be rolled back. We need to change our approach to versioning to be compatible with testnet nodes.

### How did I fix it?

- Reverted commits in https://github.com/LiskHQ/lisk/issues/2389 and https://github.com/LiskHQ/lisk/pull/2444/commits
- Changed the version number in package.json to `1.1.0`
- Fixed an issue with integration tests which was causing it to count the same node monitoring sockets multiple times after a node restarted.

### How to test it?

Run the node and connect to testnet, peer disconnects with `code 1000` should not (or rarely) happen during the sync phase.

### Review checklist

* The PR resolves #2466
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
